packages/discovery-react-components/src/components/DocumentPreview/components/HtmlView/HtmlView.tsx

```diff
@@ -53,13 +55,46 @@ export const HtmlView: FC<Props> = ({
   }
 
   const [html, setHtml] = useState<string | null>(null);
+  const [processedDoc, setProcessedDoc] = useState<ProcessedDoc | null>(null);
+  const [locationArray, setLocationArray] = useState<Array<Location> | null>(null);
```

Comment:
`Array<Location>` -> `Location[]`

Also, I think this would be better named as `highlightLocations`.